sensor_msgs/NavSatFix NavSatFix
---
nav_msgs/Odometry Odometry